misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobQueue.kt

```diff
@@ -97,6 +103,102 @@ internal class SqsJobQueue @Inject internal constructor(
     }
   }
 
+  override fun batchEnqueue(
+    queueName: QueueName,
+    jobs: List<JobQueue.JobRequest>,
+  ): JobQueue.BatchEnqueueResult {
+    return tracer.traceWithSpan("batch-enqueue-job-${queueName.value}") { span ->
+      metrics.jobsEnqueued.labels(queueName.value, queueName.value).inc(jobs.size.toDouble())
+      try {
+        check(jobs.size <= 10) {
```

Comment:
should this limit be in a constant somewhere so that callers can batch on it?

Reply:
thought of that.. but this is technically and SQS limitation. Should we enforce it at the Interface level? I think SQS is the only implementation, but if there were others this number would be implementation dependent.

Reply:
thoughts?

Reply:
added it to the interface prefixed with SQS. It's implementation specific & the sqs implementation class is internal. Not sure of a good way to do this

Reply:
I’d prefer we write code assuming exactly two implementations, SQS and a fake. If ever we add an alternate to SQS, I don’t think it’ll be a simple implementation change.